src/main/java/uk/gov/justice/hmpps/casenotes/config/OffenderCaseNoteProperties.java

```diff
@@ -23,11 +25,15 @@
 
     private final String jwtPublicKey;
 
+    private final Duration timeout;
+
     public OffenderCaseNoteProperties(@Value("${elite2.api.base.url}") @URL final String elite2ApiBaseUrl,
                                       @Value("${oauth.api.base.url}") @URL final String oauthApiBaseUrl,
-                                      @Value("${jwt.public.key}") final String jwtPublicKey) {
+                                      @Value("${jwt.public.key}") final String jwtPublicKey,
+                                      @Value("${api.ping.timeout:1s}") final Duration timeout) {
         this.elite2ApiBaseUrl = elite2ApiBaseUrl;
         this.oauthApiBaseUrl = oauthApiBaseUrl;
         this.jwtPublicKey = jwtPublicKey;
+        this.timeout = timeout;
```

Comment:
would be better to call this `healthTimeout` or `pingTimeout` to indicate not used for api calls